build.gradle

```diff
@@ -100,6 +100,19 @@ allprojects {
 
 final boolean isTravis = System.env.TRAVIS == 'true'
 
+/**
+ * Defines whether the given {@code project} should expose its JavaDoc to spine.io Web site.
```

Comment:
I also think, that that's not `spine.io` website, but `SpineEventEngine.github.io`.